src/runtime/typemanager.cs

```diff
@@ -422,6 +434,198 @@ internal static IntPtr BasicSubType(string name, IntPtr base_, Type impl)
             return type;
         }
 
+        internal class TypeSlots
```

Comment:
This should be an `enum`? Enums also have a built-in conversion from `string`.

Reply:
@lostmsu great point.  I changed it.